.github/workflows/cloud-data-sources-code-coverage.yml

```diff
@@ -0,0 +1,81 @@
+name: Cloud data sources test code coverage
+# Please be aware that this report targets cloud data source paths -- may not be as accurate as full coverage reports
+on:
+  pull_request:
+    paths:
+      - 'pkg/tsdb/azuremonitor/**'
+      - 'pkg/tsdb/cloudwatch/**'
+      - 'pkg/tsdb/cloudmonitoring/**'
+      - 'public/app/plugins/datasource/grafana-azure-monitor-datasource/**'
+      - 'public/app/plugins/datasource/cloudwatch/**'
+      - 'public/app/plugins/datasource/cloud-monitoring/**'
+    branches-ignore:
+      - dependabot/**
+      - backport-**
+
+jobs:
+  comment-coverage-difference:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - uses: actions/setup-node@v2
+        with:
+          node-version: 17.x
+          cache: yarn
+      - name: yarn install for caller's PR
+        run: yarn install
+      - uses: actions/setup-go@v2
+        with:
+          go-version: '1.17.8'
+      - name: generate go for github.com/grafana/grafana
+        run: make gen-go
+
+      - name: Get Go coverage for PR
+        run: |
+          go test -v -coverpkg=./... -coverprofile=profile.cov ./pkg/tsdb/azuremonitor/... ./pkg/tsdb/cloudmonitoring/... ./pkg/tsdb/cloudwatch/...
+          echo "go-pr-cov=$(go tool cover -func profile.cov | grep total: | awk '{print substr($NF,1,length($NF)-1)}')" >> $GITHUB_ENV
+      - name: Get TypeScript coverage for PR
+        run: |
+          echo "ts-pr-cov=$(yarn test:coverage:cloud-data-sources | grep 'All files' | awk '{print $10}')" >> $GITHUB_ENV
+      - uses: actions/checkout@v3
+        with:
+          ref: main
+      - name: yarn install for caller's main
+        run: yarn install
+      - name: generate go for github.com/grafana/grafana
+        run: make gen-go
+
+      - name: Get Go coverage on main
+        run: |
+          go test -v -coverpkg=./... -coverprofile=profile.cov ./pkg/tsdb/azuremonitor/... ./pkg/tsdb/cloudmonitoring/... ./pkg/tsdb/cloudwatch/...
+          echo "go-main-cov=$(go tool cover -func profile.cov | grep total: | awk '{print substr($NF,1,length($NF)-1)}')" >> $GITHUB_ENV
+      - name: Get TypeScript coverage on main
+        run: |
+          echo "ts-main-cov=$(yarn test:coverage:cloud-data-sources | grep 'All files' | awk '{print $10}')" >> $GITHUB_ENV
```

Comment:
I suppose you could use `yarn test:coverage -- ./public/app/plugins/datasource/cloudwatch/** ./public/app/plugins/datasource/cloud-monitoring/** ./public/app/plugins/datasource/grafana-azure-monitor-datasource/**` to avoid creating a new command in `package.json`? 
But happy either way